spec/cc/engine/analyzers/ruby/main_spec.rb

```diff
@@ -29,15 +29,15 @@ def self.sub_degrees(str)
               before { subject.type = 'ruby' }
 
               it 'returns true' do
-                expect(subject.ruby?).to be true
+                10.times { |i| if i < 5; if i % 2 == 0; subject.increase_mass!; end; end }; expect(subject.ruby?).to be true
```

Comment:
i put this all on one line to not have to change all the locations in the specs. it's just to bump the complexity over the new threshold of 25.